btr-web/btr-main-app/cypress/e2e/pages/myRegistriesDetails.cy.ts

```diff
@@ -0,0 +1,42 @@
+describe('pages -> My Registries Details', () => {
+  beforeEach(() => {
+    cy.visit('/my-registries-details')
+  })
+
+  it('rendered expected visuals', () => {
+    cy.get('[data-cy=myRegDetailsHeader]').should('contain', 'My BC Registries Details')
+    cy.get('[data-cy=myRegDetailsHeaderSub]')
+      .should('contain', 'Your BC Registries Details are shared with businesses that list you as a beneficial owner.')
```

Comment:
Nice, like this. I use this instead of contains when I stop chaining the checks.